src/com/facebook/buck/swift/SwiftCompileBase.java

```diff
@@ -167,6 +167,10 @@
   
   @AddToRuleKey private final boolean indexWhileBuilding;
 
+  @AddToRuleKey private final boolean localIndexStore;
```

Comment:
This property can't be added to the RuleKey calculation. CI is going to compile with this setting disabled and local devs with the flag enabled, and they both need to share the same artifact. You may want to add a comment here to explain our use case since it's not obvious.